bundles/org.openhab.binding.vektiva/src/main/java/org/openhab/binding/vektiva/VektivaBindingConstants.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.vektiva;
```

Comment:
One last thing: Change the `VektivaBindingConstants` file to internal and all is done.